lib/plugins/entities.js

```diff
@@ -706,3 +709,25 @@ function parseMetadata (metadata, entityMetadata = {}) {
 
   return entityMetadata
 }
+
+function extractSkinInformation (properties) {
+  if (!properties) {
+    return undefined
+  }
+
+  const skinProperty = properties.find((property) => property.name === 'textures')
```

Comment:
ok, but instead of searching you can also use a map
```js
const props = Object.fromEntries(properties.map(e) => [e.name, e])
props.textures.value ...
```